src/dbus/client/thread_api_dbus.hpp

```diff
@@ -83,22 +83,24 @@ class ThreadApiDBus
      * @param[in]   aPort     The port number.
      * @param[in]   aSeconds  The timeout to close the port, 0 for never close.
      *
-     * @retval OT_ERROR_NONE successfully performed the dbus function call
-     * @returns Error value otherwise
+     * @retval ERROR_NONE successfully performed the dbus function call
+     * @retval ERROR_DBUS dbus encode/decode error
+     * @returns OpenThread defined error value otherwise
```

Comment:
Sorry, I don't think I'm being clear. I want to avoid mixing `@retval` and `@returns` in the same block. If we want to call out a couple explicit values as you have done, I'd advocate:
```
     * @retval ERROR_NONE successfully performed the dbus function call
     * @retval ERROR_DBUS dbus encode/decode error
     * @retval other      OpenThread defined error value otherwise
```